reference-architecture/vmware-ansible/roles/docker-storage-setup/tasks/main.yaml

```diff
@@ -0,0 +1,13 @@
+---
+- name: create the docker-storage-setup config file
+  copy:
+     src: docker-storage-setup
+     dest: /etc/sysconfig/docker-storage-setup
```

Comment:
Might want to consider making docker-storage-setup a template and allowing the user to override the settings.